doc/neomake.txt

```diff
@@ -31,13 +31,13 @@ fantastic vim plugins such as syntastic and dispatch.
 ==============================================================================
 2. Commands                                                   *neomake-commands*
 
-                                                         *:Neomake* *:NeomakeFile*
+                                                       *:Neomake* *:NeomakeFile*
```

Comment:
I intentionally moved it to line up with the other helptag names -- they're right aligned to the same column with noconceal